entity-framework/core/extensions/index.md

```diff
@@ -27,6 +27,12 @@ Entity Developer is a powerful ORM designer for ADO.NET Entity Framework, NHiber
 
 [Website](https://www.devart.com/entitydeveloper/)
 
+### nHydrate ORM for Entity Framework
+
+An ORM that creates strongly-typed, extendable classes for Entity Framework. The generated code is Entity Framework Core. There is no difference. This is not a replacement for EF or a custom ORM. It is a visual, modeling layer that allows a team to manage complex database schemas. It works well with SCM software like Git, allowing multi-user access to your model with minimal conflicts. The installer tracks model changes and creates upgrade scripts.
```

Comment:
@gravbox Please indicate which major versions of EF Core this works with, like for the other tools here.

Reply:
I have added "For EF Core: 3." now. Thanks.